README.markdown

```diff
@@ -206,6 +206,40 @@ bool2str(false, 't', 'f')         => 'f'
 
 Requires a single boolean as input. *Type*: rvalue.
 
+#### `bourne_shell_escape`
```

Comment:
Could you actually just name these `shell_*` instead of `bourne_shell_*` ? It's kind of assumed that unless explicitly mentioned like "ksh", "csh", "bash" that "shell" means "bourne".